wesnoth-sof.po

```diff
@@ -2415,17 +2417,12 @@ msgstr "Máš už něco?!"
 
 #. [message]: speaker=Theganli
 #: data/campaigns/Sceptre_of_Fire/scenarios/4t_The_Jeweler.cfg:295
-#, fuzzy
-#| msgid ""
-#| "Ah, well, uh, no... no, not yet... unfortunately... it seems... it can’t "
-#| "be cut... or scratched... or damaged at all... at least not by my tools..."
 msgid ""
 "Ah, well, uh, no... no... unfortunately... It seems it can’t be cut, or "
 "scratched, or damaged at all! At least not by my tools..."
 msgstr ""
-"Ah, no, uh, ne, ještě ne... naneštěstí... zdá se... že nemůže být "
-"zbroušen... nebo poškrábán... nebo vůbec poškozen... alespoň ne mými "
-"nástroji."
+"Ah, no, uh, ne... ne... naneštěstí... Zdá se..., že nemůže být zbroušen... "
+"nebo poškrábán... nebo vůbec poškozen... alespoň ne mými nástroji..."
```

Comment:
opravdu tam chcete nechat tolik teček, když se jich v předloze vzdali?
```suggestion
"Ah, no, uh, ne... ne... naneštěstí... Zdá se, že nemůže být zbroušen, "
"nebo poškrábán, nebo vůbec poškozen! Alespoň ne mými nástroji..."
```

Reply:
Aha. Já tam ty tečky v originále při překládání fakt viděl! To byla tedy hustá vlčí mlha.